src/main/java/no/unit/nva/model/instancetypes/ReportResearch.java

```diff
@@ -0,0 +1,42 @@
+package no.unit.nva.model.instancetypes;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import no.unit.nva.model.exceptions.InvalidPageTypeException;
+import no.unit.nva.model.pages.Pages;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class ReportResearch extends Report {
```

Comment:
Are a ReportResearch and a ReportPolicy if they share the same values in their fields?

Reply:
They are equal as of now, but I believe they will differ when we know more about the types.